back/src/main/java/fr/formation/itschool/masterpiece/services/CountryServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package fr.formation.itschool.masterpiece.services;
+
+import fr.formation.itschool.masterpiece.dtos.CountryViewDto;
+import fr.formation.itschool.masterpiece.repositories.CountryRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CountryServiceImpl implements CountryService {
+
+  private final CountryRepository countryRepository;
+  private final ModelMapper modelMapper;
```

Comment:
modelmapper in collaboratorServiceImpl, model**M**apper here. Please keep the same naming conventions everywhere